modules/Total/index.js

```diff
@@ -0,0 +1,28 @@
+const getEmote = (emote, discord) => {
+  return discord ? `:${emote}:` : emote;
+};
+
+const getBold = (string, discord) => {
+  return discord ? `**${string}**` : string;
+};
+
+module.exports = {
+  name: 'Total',
+  module(jaffamod) {
+    jaffamod.registerCommand('total', (message, reply, discord) => {
+      // Determine if it's JingleJam time
+      //  (December + first 7 days of the new year to view the total raised)
+      const d = new Date();
+      if (d.getMonth() === 11 || (d.getMonth() === 0 && d.getDate() <= 7)) {
+        jaffamod.api.get('https://jinglejam.yogscast.com/api/total').catch(() => {
+          reply(`The total amount couldn't be determined currently. ${getEmote('yogP3', discord)} Please try again later.`);
```

Comment:
I wrote a little helper util in this module to correctly return an emote (Discord wraps with `:`, Twitch doesn't). Just wanted to check nothing like this exists in the bot already?